### PR TITLE
Clear input value

### PR DESCRIPTION
**Pull request by slauzinho**
Can we also clear the input value if we call `clearSearch` since it implies that we are clearing the search input.

### CODE REVIEW

**Comment by EQuimper** [comment]
@slauzinho good idea, but your input is surely already control from your side. And not everyone want this behavior I think. Maybe I can add a boolean option where this can do both. What you think?

**Comment by slauzinho** [comment]
Sounds good to me, I can do it for you if you want. Should I add it as a prop of the component or a param for the `clearSearch` which defaults to not clearing the text field?

**Comment by EQuimper** [comment]
I think the component is best, but I'm open to anything

File: lib/GoogleAutocomplete.tsx [hunk 1]
```diff
@@ -39,7 +39,12 @@ export interface DefaultProps {
   minLength: number;
   debounce: number;
   language: string;
-  queryTypes: 'address' | 'geocode' | 'cities' | 'establishment' | 'geocode|establishment';
```
Comment (EQuimper) [inline lib/GoogleAutocomplete.tsx]: I dont think this change is need can you rollback this one

File: lib/GoogleAutocomplete.tsx [hunk 2]
```diff
@@ -39,7 +39,12 @@ export interface DefaultProps {
...
+  queryTypes:
```
Comment (EQuimper) [inline lib/GoogleAutocomplete.tsx]: I dont think this change is need can you rollback this one

Comment (slauzinho) [reply lib/GoogleAutocomplete.tsx]: Will work on submitting an updated PR this weekend

Comment (EQuimper) [reply lib/GoogleAutocomplete.tsx]: Perfect thank you :)

**Comment by karimcambridge** [comment]
@slauzinho did you forget about this?

**Comment by juanlet** [comment]
what happened to this change? Any news? It would be great to have that boolean as a parameter to clearSearch....really handy

**Comment by EQuimper** [comment]
This will now be in v 2 by default thank you

**Pull request closed without merging**